privacy-security-questionnaire.md

```diff
@@ -0,0 +1,97 @@
+ * What potentially identifying information does your spec expose to third parties that third parties can already access: None.
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+Yes. The feature only exposes the minimum required information, as a boolean header, to satisfy the use case.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+No. The feature does not relate to PII or any derived information thereof.
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+No. The feature does not deal with sensitive information.
+
+> 05.  Does data exposed by your specification carry related but distinct
+>      information that may not be obvious to users?
+
+Yes. The specification carries related information about the page, specifically if there are linked resources to be expected in the requested page, as a Document-Policy header in the response header, which will not be obvious to users.
+
+> 06.  Do the features in your specification introduce state
+>      that persists across browsing sessions?
+
+No. The feature in the specification only applies to a single response and does not persist across browsing session.
+
+> 07.  Do the features in your specification expose information about the
+>      underlying platform to origins?
+
+No.
+
+> 08.  Does this specification allow an origin to send data to the underlying
+>      platform?
+
+No.
+
+> 09.  Do features in this specification enable access to device sensors?
+
+No.
+
+> 10.  Do features in this specification enable new script execution/loading
+>      mechanisms?
+
+No.
+
+> 11.  Do features in this specification allow an origin to access other devices?
+
+No.
+
+> 12.  Do features in this specification allow an origin some measure of control over
+>      a user agent's native UI?
+
+No.
+
+> 13.  What temporary identifiers do the features in this specification create or
+>      expose to the web?
+
+None.
+
+> 14.  How does this specification distinguish between behavior in first-party and
+>      third-party contexts?
+
+It does not differentiate.
+
+> 15.  How do the features in this specification work in the context of a browser’s
+>      Private Browsing or Incognito mode?
+
+It does not differentiate.
+
+> 16.  Does this specification have both "Security Considerations" and "Privacy
+>      Considerations" sections?
+
+No. The feature in the specification does not have any security or privacy impact.
+
+> 17.  Do features in your specification enable origins to downgrade default
+>      security protections?
+
+No.
+
+| 2.17. How does your feature handle non-"fully active" documents?
```

Comment:
Questions 18-21 of the template don't match this.

I guess the difference is https://w3ctag.github.io/security-questionnaire/ (latest version) vs. https://www.w3.org/TR/security-privacy-questionnaire/ (older version). Ugh. And neither perfectly matches the markdown template.

I say stick with the Markdown template, and I'll file an issue about this mismatch.

Reply:
There is some difference in questions, yes. I updated to stick with the template.